### PR TITLE
Small clothing PR (ezpz 2 review & merge)

### PR DESCRIPTION
**Pull request by KathrinBailey**
## About The Pull Request
• Fixes jean skirts/shorts covering body parts it shouldn't.
• Adds in a skirt to the clothing vendor.
• Adds in loincloths from our upstream's old upstream's downstream and makes them craftable, this is for our upstream's old upstream's downstream's downstream who will be rebasing to become our new downstream in a few months, hopefully.

## Why It's Good For The Game
Bugs bad.
Making clothing accessible whilst loadouts are broken good.
Tribal clothing for tribals are good.

## Proof Of Testing
image
image
image
image

## Changelog

:cl:
add: Loincloths from Sandstorm.
fix: Jean shorts/skirt no longer covers legs and chest.
/:cl:

### CODE REVIEW

File: modular_skyrat/master_files/code/modules/clothing/under/skirts_dresses.dm [hunk 1]
```diff
@@ -88,6 +88,7 @@
 	greyscale_config_worn_digi = /datum/greyscale_config/jean_skirt/worn/digi
 	greyscale_colors = "#787878#723E0E#4D7EAC"
 	flags_1 = IS_PLAYER_COLORABLE_1
+	body_parts_covered = GROIN
```
Comment (projectkepler-ru) [inline modular_skyrat/master_files/code/modules/clothing/under/skirts_dresses.dm:91]: this specific part should be done modularly, or, upstream

**Review by projectkepler-ru — CHANGES_REQUESTED** [review]

File: modular_skyrat/master_files/code/modules/clothing/under/skirts_dresses.dm (hunk 1 above)
Comment (KathrinBailey) [reply modular_skyrat/master_files/code/modules/clothing/under/skirts_dresses.dm:91]: https://github.com/Skyrat-SS13/Skyrat-tg/pull/29598

File: modular_zubbers/code/modules/vending/lustwish.dm [hunk 2]
```diff
@@ -18,6 +18,8 @@
 				/obj/item/clothing/neck/tie/bunnytie = 6,
 				/obj/item/clothing/under/costume/playbunny = 6,
 				/obj/item/clothing/suit/jacket/tailcoat = 6,
+				/obj/item/clothing/under/costume/loincloth/sensor = 6,
+				/obj/item/clothing/under/costume/loincloth/cloth/sensor = 6,
```
Comment (StrangeWeirdKitten) [inline modular_zubbers/code/modules/vending/lustwish.dm:22]: Too ambuguous. It'll just be the same item. Just have the vendors dispense the sensor versions while the crafting recipe doesn't have sensors.

**Review by StrangeWeirdKitten — CHANGES_REQUESTED** [review]

File: modular_zubbers/code/modules/vending/lustwish.dm (hunk 2 above)
Comment (KathrinBailey) [reply modular_zubbers/code/modules/vending/lustwish.dm:22]: But the vendors already dispense the sensor versions, and the crafting menu already crafts the no sensor versions?

**Pull request merged**